Solution1/ClassLibrary1/NumericUpDownLight.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace ClassLibrary1
+{
+    public class NumericUpDownLight : Control
+    {
+        private int numericValue = 0;
+        private HorizontalAlignment alignmentValue = HorizontalAlignment.Left;
+        private Color mouseHoverBackColorValue = Color.Blue;
+
+        private CustomTextBox TextBox = new CustomTextBox();
```

Comment:
Dominic, do you know how a field differs from a property and a variable?

Reply:
yes, of course